docs/samples/explanation/alibi/moviesentiment/kfserving_text_explainer.ipynb

```diff
@@ -0,0 +1,252 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Sample for KFService Explainer"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The notebook shows how to use KFServing SDK to create KFService with explainer, predictor."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 59,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from kubernetes import client\n",
+    "\n",
+    "from kfserving import KFServingClient\n",
+    "from kfserving import constants\n",
+    "from kfserving import V1alpha2EndpointSpec\n",
+    "from kfserving import V1alpha2PredictorSpec\n",
+    "from kfserving import V1alpha2ExplainerSpec\n",
+    "from kfserving import V1alpha2AlibiExplainerSpec\n",
+    "from kfserving import V1alpha2SKLearnSpec\n",
+    "from kfserving import V1alpha2KFServiceSpec\n",
+    "from kfserving import V1alpha2KFService\n",
+    "from kubernetes.client import V1ResourceRequirements\n",
+    "from kubernetes import client, config\n",
+    "import kubernetes.client\n",
+    "import os\n",
+    "import requests \n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Define KFService with Explainer"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Add predictor and explainer on the endpoint spec"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "api_version = constants.KFSERVING_GROUP + '/' + constants.KFSERVING_VERSION\n",
+    "default_endpoint_spec = V1alpha2EndpointSpec(\n",
+    "                          predictor=V1alpha2PredictorSpec(\n",
+    "                            min_replicas=1,\n",
+    "                            sklearn=V1alpha2SKLearnSpec(\n",
+    "                              storage_uri='gs://seldon-models/sklearn/moviesentiment',\n",
+    "                              resources=V1ResourceRequirements(\n",
+    "                                  requests={'cpu': '100m', 'memory': '1Gi'},\n",
+    "                                  limits={'cpu': '100m', 'memory': '1Gi'}))),\n",
+    "                          explainer=V1alpha2ExplainerSpec(\n",
+    "                            min_replicas=1, \n",
+    "                            alibi=V1alpha2AlibiExplainerSpec(\n",
+    "                              type='AnchorText',\n",
+    "                              resources=V1ResourceRequirements(\n",
```

Comment:
I think creating an AnchorsText saved to a bucket might be better in the long run so the initContainer can be used. The main issue is the download of the Spacy corpus which is big. I might try to update to see if it works.